x-pack/plugins/security_solution/public/explore/hosts/components/uncommon_process_table/index.tsx

```diff
@@ -8,17 +8,14 @@
 import React, { useCallback, useMemo } from 'react';
 import { useDispatch } from 'react-redux';
 
-import type { HostEcs } from '@kbn/securitysolution-ecs';
 import type { HostsUncommonProcessesEdges } from '../../../../../common/search_strategy';
 import { hostsActions, hostsModel, hostsSelectors } from '../../store';
-import { defaultToEmptyTag, getEmptyValue } from '../../../../common/components/empty_value';
-import { HostDetailsLink } from '../../../../common/components/links';
 import type { Columns, ItemsPerRow } from '../../../components/paginated_table';
 import { PaginatedTable } from '../../../components/paginated_table';
 import * as i18n from './translations';
-import { getRowItemsWithActions } from '../../../../common/components/tables/helpers';
 import { HostsType } from '../../store/model';
 import { useDeepEqualSelector } from '../../../../common/hooks/use_selector';
+import { getUncommonColumns } from './columns';
```

Comment:
Consider importing `UncommonProcessTableColumns` type as well, right now it is duplicated.
```suggestion
import { getUncommonColumns, type UncommonProcessTableColumns } from './columns';
```